Game.py

```diff
@@ -156,7 +160,7 @@ def _process_ability(self, organism):
 			organism.change_num_mana(mana_stolen)
 
 		# Increasing move count on next turn
-		if organism.ability.increase_move_count:
-			self._curr_player.give_extra_move(False)
+		if organism.ability.move_bonus:
+			self._curr_player.give_extra_move(False, organism.ability.move_bonus, organism.ability.move_bonus_duration)
```

Comment:
Had a good idea (I think). Keep give extra move, but dont overload it. So give_extra_move() takes no arguments and provides the move during the current turn if possible. Rename "add_bonus_move_delta" to "change_num_moves" for consistency with the other exposed change functions. And just call "change_num_moves" here directly.